src/models/config.py

```diff
@@ -88,7 +89,10 @@ def __init__(self, path):
         def __read_data():
             with open(self.path, 'r') as stream:
                 return yaml.load(stream) or {}
-        self.data = __read_data()
+        try:
+            self.data = __read_data()
+        except FileNotFoundError:
```

Comment:
Typically it is best practice to check if the file exists up front instead of relying on `try-catch`. It amounts to the same thing, however it doesn't try and read a non-existent file.

Reply:
sure, done here: https://github.com/alces-software/adminware/pull/160/commits/48dbebe700777433e9d95e27f78604ee41319b52